tests/hassfest/test_translations.py

```diff
@@ -19,3 +23,393 @@ def test_string_with_no_placeholders_in_single_quotes() -> None:
+                    "low": "Low",
+                    "high": "High",
+                },
+                "state_attributes": {
+                    "trend": {
+                        "name": "Trend",
+                        "state": {
+                            "rising": "Rising",
+                            "falling": "Falling",
+                        },
+                    },
+                },
+                "unit_of_measurement": "degrees",
+            },
+            "humidity": {
+                "name": "Humidity",
+            },
+        },
+        "binary_sensor": {
+            "motion": {
+                "name": "Motion",
+                "state": {
+                    "on": "Detected",
+                    "off": "Clear",
+                },
+            },
+        },
+        "switch": {
+            "power": {
+                "name": "Power",
+            },
+        },
+    },
+    "exceptions": {
+        "cannot_connect": {
+            "message": "Cannot connect to {host}",
+        },
+        "invalid_credentials": {
+            "message": "Invalid credentials for {username}",
+        },
+    },
+    "services": {
+        "set_value": {
+            "name": "Set value",
+            "description": "Sets a value on the device",
+            "fields": {
+                "value": {
+                    "name": "Value",
+                    "description": "The value to set",
+                    "example": "100",
+                },
+                "target": {
+                    "name": "Target",
+                    "description": "The target device",
+                },
+            },
+            "sections": {
+                "advanced": {
+                    "name": "Advanced options",
+                    "description": "Advanced configuration options",
+                },
+            },
+        },
+        "reload": {
+            "name": "Reload",
+            "description": "Reloads the integration",
+        },
+    },
+    "conditions": {
+        "is_on": {
+            "name": "Is on",
+            "description": "Test if the device is on",
+            "fields": {
+                "entity_id": {
+                    "name": "Entity",
+                    "description": "The entity to check",
+                    "example": "light.living_room",
+                },
+            },
+        },
+    },
+    "triggers": {
+        "turned_on": {
+            "name": "Turned on",
+            "description": "When the device turns on",
+            "fields": {
+                "entity_id": {
+                    "name": "Entity",
+                    "description": "The entity to monitor",
+                    "example": "light.living_room",
+                },
+            },
+        },
+    },
+    "conversation": {
+        "agent": {
+            "done": "Done",
+        },
+    },
+    "common": {
+        "on": "On",
+        "off": "Off",
+        "unknown": "Unknown",
+    },
+}
```

Comment:
We could consider moving this to a fixture.

Also, I would suggest to sort this to make it more similar to the snapshot (which is alphabetically sorted)

Reply:
I removed snapshot assertion upon suggestion by Erik.